containers/ecr-viewer/integration/listEcrDataService.test.tsx

```diff
@@ -0,0 +1,420 @@
+      process.env.METADATA_DATABASE_TYPE = "postgres";
+    });
+
+    describe("listExtendedEcrData", () => {
+      it("should return data when found", async () => {
+        // Arrange
+        const insert = await (db as any)
+          .insertInto("ecr_viewer.ecr_data")
+          .values({
+            eicr_id: "12345",
+            set_id: "123",
+            data_source: "DB",
+            fhir_reference_link: "",
+            eicr_version_number: "1",
+            patient_name_first: "Billy",
+            patient_name_last: "Bob",
+            patient_birth_date: new Date("2024-12-01T12:00:00Z"),
+            date_created: new Date("2024-12-01T12:00:00Z"),
+            report_date: new Date("2024-12-01T12:00:00Z"),
+          })
+          .returningAll()
+          .executeTakeFirstOrThrow();
+        const mockRecordset = [
+          {
+            eICR_ID: "123",
+            first_name: "John",
+            last_name: "Doe",
+            birth_date: new Date("1990-01-01"),
+            encounter_start_date: new Date("2023-01-01T07:30:00Z"),
+            date_created: new Date("2023-01-02T07:45:00Z"),
+            conditions: "Condition1,Condition2",
+            rule_summaries: "Rule1,Rule2",
+            set_id: "123",
+            eicr_version_number: "1",
+          },
+          {
+            eICR_ID: "124",
+            first_name: "Jane",
+            last_name: "Doe",
+            birth_date: new Date("1990-01-02"),
+            encounter_start_date: new Date("2023-01-02T07:30:00Z"),
+            date_created: new Date("2023-01-01T07:45:00Z"),
+            conditions: "Condition1,Condition2",
+            rule_summaries: "Rule1,Rule2",
+          },
+        ];
+
+        // Act
+        const result = await listEcrData(
+          0,
+          10,
+          "report_date",
+          "DESC",
+          testDateRange,
+        );
+
+        // Assert
+        expect(result).toEqual([
+          {
+            ecrId: "12345",
+            patient_first_name: "Billy",
+            patient_last_name: "Bob",
+            patient_date_of_birth: "12/01/2024",
+            reportable_conditions: [null],
+            rule_summaries: [null],
+            date_created: "12/01/2024 7:00\u00A0AM\u00A0EST",
+            patient_report_date: "12/01/2024 12:00\u00A0AM\u00A0EST",
+            eicr_set_id: "123",
+            eicr_version_number: "1",
+          },
+        ]);
+      });
+    });
+
+    describe("getTotalEcrCount with SQL Server", () => {
+      it("should return count when DATABASE_TYPE is sqlserver", async () => {
+        // Act
+        const count = await getTotalEcrCount(testDateRange);
+
+        // Assert
+        expect(count).toEqual("1");
+      });
+    });
+  });
+
+  describe("get total ecr count", () => {
+    beforeAll(async () => {
+      process.env.METADATA_DATABASE_TYPE = "postgres";
+      await buildCoreAlias();
+    });
+    afterAll(async () => {
+      delete process.env.METADATA_DATABASE_TYPE;
```

Comment:
instead of deleting, can you follow the pattern in unit tests and do
```
const ORIG_METADATA_DATABASE_TYPE = process.env.METADATA_DATABASE_TYPE
```
then set it back to that in the after. Typescript doesn't typically like the deleting of env vars that we've told it exist